app/client/pages/course/courseSummary.js

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+
+
+
+const post = {
```

Comment:
for future: path should not be -> `/course/courseSummary.js`, it should be `/course/[course_id].js`
Then you can get course_id:
```
const router = useRouter();
const { course_id } = router.query;
```
You can then make request to server to fetch data for the page.